docs/preliminaries.adoc

```diff
@@ -7,9 +7,34 @@ To use *Arquillian Cube* you need a _Docker_ daemon running on a computer (it ca
 By default the _Docker_ server uses UNIX sockets for communicating with the _Docker_ client. *Arquillian Cube* will attempt to detect the operating system it is running on and either set _docker-java_ to use UNIX socket on _Linux_ or to <<Boot2Docker>> on _Windows_/_Mac_ as the default URI.
 
 If you want to use TCP/IP to connect to the Docker server, you'll need to make sure that your _Docker_ server is listening on TCP port.
-To allow _Docker_ server to use TCP add the following line to +/etc/default/docker+:
+To allow _Docker_ server to use TCP set the _Docker daemon options_, the exact process for which varies by the way you
```

Comment:
I'd add a clarification that if you are using Docker-Machine this is not necessary, and also that Cube works with native sockets, and these steps are only required if you want to communicate through TCP.

Reply:
Isn't this same as: 

> *Arquillian Cube* will attempt to detect the operating system it is running on and either set _docker-java_ to use UNIX socket on _Linux_ or to <<Boot2Docker>> on _Windows_/_Mac_ as the default URI.

Reply:
@lordofthejars, Does Docker-Machine also take care of TCP communication by default?

Reply:
yes, in case of docker machine everything is configured automatically.